test/e2e/app-dir/metadata-streaming/metadata-streaming.test.ts

```diff
@@ -88,6 +88,17 @@ describe('app-dir - metadata-streaming', () => {
     expect((await browser.elementsByCss('body meta')).length).toBe(9)
   })
 
+  it('should only insert metadata once for parallel routes', async () => {
+    const browser = await next.browser('/parallel-routes')
+
+    expect((await browser.elementsByCss('head title')).length).toBe(1)
+    expect((await browser.elementsByCss('body title')).length).toBe(0)
+
+    const html = await next.render('/parallel-routes')
+    // only one title tag once in the html
+    expect(html.split('<title>').length).toBe(2)
```

Comment:
the comment here doesn't seem to match the assertion - is this because the second one is in the RSC payload?

Reply:
The RSC pay load won't have `"<>"` but I just change assertion to use cheerio for better readability